src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,21 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.figures.Figure;
+
 public class HelloWorld {
+    private static FigureSupplier figureSupplier = new FigureSupplier();
 
+    public static void main(String[] args) {
+        Figure[] figures = {
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getDefaultFigure(),
+                figureSupplier.getDefaultFigure(),
+                figureSupplier.getDefaultFigure()
+        };
```

Comment:
You need to create a loop that will add a figure to the `figures` array. Additionally, you should add a check to determine whether it is the first or second half of the array. If it is the first half, you should add a `randomFigure`, and if it is the second half, you should add a `defaultFigure`.